.github/ISSUE_TEMPLATE/feature_request.yaml

```diff
@@ -0,0 +1,37 @@
+name: Feature Request
+description: Suggest an idea for this project
+title: "[Feature Request]: <title>"
+labels: []
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
```

Comment:
this isn't a bug report

Reply:
[Fixed](https://github.com/JaydenDev/Catalyst/pull/180/commits/42d0a058965bdabd04382cf88cdfc5ca7ff12a22)